lib/rules/no-multi-spaces.js

```diff
@@ -7,191 +7,42 @@
 "use strict";
```

Comment:
I just noticed that your name doesn't appear here, even though you completely rewrote the rule.